### PR TITLE
Support for new os/devices, CTS

### PR DESCRIPTION
**Pull request by OSIRIS-REx**
Added support for new os/devices from CTS (Connection Technology Systems).
Some serial and sensors support added for certain models.

LibreNMS CTS screenshot

Discovery and polling hangs and fails when using snmpbulkwalk. Manually running snmpbulkwalk also fails after some seconds, the device simply stops responding. `snmp_bulk` is therefored disabled.

These devices seem to have a strange SNMP implementation of the fdb-table, and the discovery is throwing out errors. I couldn't find a way to retrieve the FDB-table correctly. Therefore the fdb-table discovery module is disabled.
```
Error discovering fdb-table module for <ip-address>. TypeError: Illegal offset type in /opt/librenms-dev/includes/discovery/fdb-table/bridge.inc.php:83
Stack trace:
#0 /opt/librenms-dev/includes/discovery/fdb-table.inc.php(29): include()
#1 /opt/librenms-dev/includes/discovery/functions.inc.php(164): include('/opt/librenms-d...')
#2 /opt/librenms-dev/discovery.php(108): discover_device()
#3 {main}  
SQL[insert into `eventlog` (`reference`, `type`, `datetime`, `severity`, `message`, `username`, `device_id`) values (?, ?, ?, ?, ?, ?, ?) [null,"discovery","2024-01-30 11:35:50",5,"Error discovering fdb-table module. Check log file for more details.","",9] 0.44ms] 
  
Illegal offset type {"exception":"[object] (TypeError(code: 0): Illegal offset type at /opt/librenms-dev/includes/discovery/fdb-table/bridge.inc.php:83)"} 
```

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [X] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [X] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/cts.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    value: IPS-3110-MIB::swhSFPPortStateTemperature
+                    num_oid: '.1.3.6.1.4.1.9304.100.3110.5.34.2.1.1.5.{{ $index }}'
+                    descr: '{{ IPS-3110-MIB::swhSFPPortStatePortName }} SFP Temperature'
+                    index: 'ddmTemp-{{ $index }}'
+                    group: '{{ IPS-3110-MIB::swhSFPPortStatePortName }}'
+        power:
+            data:
+                -
+                    oid: IPS-3110-MIB::swhPOETotalPower
+                    num_oid: '.1.3.6.1.4.1.9304.100.3110.5.48.1.0'
```
Comment (PipoCanaja) [inline includes/definitions/discovery/cts.yaml:53]: should be `num_oid: '.1.3.6.1.4.1.9304.100.3110.5.48.1.{{ $index }}'`

**Review by PipoCanaja — CHANGES_REQUESTED** [review]
Hi,
You have a typo that prevents the tests from succeeding

File: includes/definitions/discovery/cts.yaml (hunk 1 above)
Comment (OSIRIS-REx) [reply includes/definitions/discovery/cts.yaml]: Hi, thank you. I wasn't aware that {{ $index }} was needed here as well. I've fixed it now and pushed the changes.

**Review by Jellyfrog — APPROVED** [review]

**Comment by Jellyfrog** [comment]
Congrats on your first pr @OSIRIS-REx

**Pull request merged**